### PR TITLE
Validate scientific extension, update for #1045 (merge after scientific extension 1.0.0 release)

### PR DESCRIPTION
**Pull request by cholmes**
**Related Issue(s):** #1093 https://github.com/stac-extensions/scientific/pull/2

**Proposed Changes:**

1. Bring in the updated scientific example, so we can release rc.3 without waiting on  https://github.com/stac-extensions/scientific/pull/2, as discussed in https://github.com/radiantearth/stac-spec/pull/1093/files#r620248117

**PR Checklist:**

- [x] This PR is made against the dev branch (all proposed changes except releases should be against dev, not master).
- [x] This PR has **no** breaking changes.
- [x]  a CHANGELOG entry is not required.

### CODE REVIEW

**Comment by m-mohr** [comment]
I don't understand this PR? Do we remove the example for RC3 through #1104 and then add it back again after RC3? In this case the Schema URL needs to update from 1.0.0 to 1.0.1.

**Comment by cholmes** [comment]
I thought that's what you were intending, but I wasn't sure, so happy to take another route.

My take was we should try to release rc.3 asap, and if it's an example of an extension that is blocking us then we should just take that out. But I think it's a good example, so whenever it's in good shape then we should merge it back in.

But if it can happen today/tomorrow I'm for just merging in the scientific fix and cutting 1.0.1 from there as the route to unblock this.

**Comment by m-mohr** [comment]
So this PR is not meant to be merged for RC3, understood. Then we need to update the schema URI, of course.

File: examples/collection-only/collection.json [hunk 1]
```diff
@@ -1,7 +1,9 @@
 {
   "type": "Collection",
   "stac_version": "1.0.0-rc.2",
-  "stac_extensions": [],
+  "stac_extensions": [
+    "https://stac-extensions.github.io/scientific/v1.0.0/schema.json"
```
Comment (m-mohr) [inline examples/collection-only/collection.json:5]:
```suggestion
    "https://stac-extensions.github.io/scientific/v1.0.1/schema.json"
```

**Comment by m-mohr** [comment]
Looking at it again, I think we could simply not merge it and have these examples in the corresponding extension repos. It's nothing major anyway.

**Pull request closed without merging**